_team/aung.md

```diff
@@ -2,10 +2,11 @@
 firstname: "Amanda"
 lastname: "Ung"
 group: "member"
+graduating_year: 2023
+github: "a17ung"
 title: "Emerson Instructor"
 secondary: "Developer"
 pronouns: "she/her"
 img: "aung.jpg"
```

Comment:
Think you accidentally deleted the old image and kept the artifact (`aung.jpg.orig`) - you may want to revert that so your team page works!